poise/hello-world/src/lib.rs

```diff
@@ -0,0 +1,42 @@
+use anyhow::Context as _;
```

Comment:
```suggestion
use anyhow::Context;
```

Reply:
`anyhow::Context` isn't used directly (only via `.context()`) and otherwise conflicts with the Context type [you make for Poise](https://github.com/Anafabula/shuttle-examples/blob/87d9a6146311b538ca026e8eee709d35096d299e/poise/hello-world/src/lib.rs#L8), but it would be trivial to rename that one to something like MyContext.
The Postgres example for serenity [uses `use anyhow::Context as _;`](https://github.com/shuttle-hq/examples/blob/main/serenity/postgres/src/lib.rs#L1) as well.

Reply:
Ahh, I missed the poise `Context` type